ckan/config/environment.py

```diff
@@ -65,7 +65,7 @@ def load_environment(conf: Union[Config, CKANConfig]):
         warnings.filterwarnings('ignore', msg, sqlalchemy.exc.SAWarning)
 
     # load all CKAN plugins
-    p.load_all()
+    p.load_all(force_update=True)
```

Comment:
Why is force_update required here?

Reply:
load_environment is used in tests which sometimes need to clear all the plugins by having an empty plugin list